payment-failure/src/main/scala/com/gu/identity/paymentfailure/abtest/EncryptedEmailTest.scala

```diff
@@ -0,0 +1,38 @@
+package com.gu.identity.paymentfailure.abtest
+
+import cats.syntax.either._
+import com.gu.identity.paymentfailure.{BrazeClient, IdentityClient}
+import com.gu.identity.paymentfailure.IdentityClient.IdentityEmailTokenRequest
+
+import scala.util.Random
+
+class EncryptedEmailTest(identityClient: IdentityClient) extends VariantGenerator {
```

Comment:
Previous test, refactored to implement the `VariantGenerator` trait. For now, keeping this file so we have a record-in-code of the tests run.